chunky/src/java/se/llbit/math/PackedOctree.java

```diff
@@ -163,30 +202,31 @@ private int findSpace() {
     }
 
     // We need to grow the array
-    long newSize = (long) Math.ceil(treeData.length * 1.5);
+    long newSize = (long) Math.ceil(treeData.length * ARRAY_RESIZE_MULTIPLIER);
     // We need to check the array won't be too big
     if(newSize > (long) MAX_ARRAY_SIZE) {
       // We can allocate less memory than initially wanted if the next block will still be able to fit
       // If not, this implementation isn't suitable
-      if(MAX_ARRAY_SIZE - treeData.length > 8) {
+      if(MAX_ARRAY_SIZE - size > 8) {
```

Comment:
Note: there might be a small chance we can fit in a node between the end of used array (size), and MAX_ARRAY_SIZE, which would be missed if split by treeData.length.